packages/release-manifests/report.api.md

```diff
@@ -21,6 +21,12 @@ export function getManifestByVersion(
 // @public
 export type GetManifestByVersionOptions = {
   version: string;
+  fetchFn?: (
```

Comment:
```suggestion
  fetch?: (
```

since fetch is already a function I'm thinking that's enough

Reply:
Updated :+1: